Keas.Mvc/Services/ReportService.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Keas.Core.Data;
+using Keas.Core.Domain;
+using Keas.Mvc.Models.ReportModels;
+using Microsoft.EntityFrameworkCore;
+
+namespace Keas.Mvc.Services
+{
+    public interface IReportService
+    {
+        Task<IList<WorkstationReportModel>> WorkStations(Team team, string teamSlug);
+    }
+
+
+    public class ReportService : IReportService
+    {
+        private readonly ApplicationDbContext _context;
+
+        public ReportService(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+
+
+        public async Task<IList<WorkstationReportModel>> WorkStations(Team team, string teamSlug)
+        {
+            if (team == null)
+            {
+                team = await _context.Teams.SingleAsync(a => a.Slug == teamSlug);
+            }
+
+            return await _context.Workstations.Include(a => a.Space).Include(a => a.Assignment).ThenInclude(a => a.Person).IgnoreQueryFilters().AsNoTracking().Where(a => a.TeamId == team.Id).Select(a => new WorkstationReportModel

```

Comment:
try getting rid of the `.Include(...)` parts of this and run again.  I don't think you'll need them and I think they might be grabbing more info than you need.  You don't need the `.Include()` if you are projecting out with a `.Select()`